.github/workflows/images.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         os=$(go env GOOS)
         arch=$(go env GOARCH)
         # download kubebuilder and extract it to tmp
-        curl -L https://go.kubebuilder.io/dl/2.3.1/${os}/${arch} | tar -xz -C /tmp/
+        curl -L https://github.com/kubernetes-sigs/kubebuilder/releases/download/v2.3.1/kubebuilder_2.3.1_${os}_${arch}.tar.gz | tar -xz -C /tmp/
```

Comment:
we shouldn't hardcode the version here (or, if we do, we should pull in patch and minor updates automatically)